docs/UPDATE_CORE.md

```diff
@@ -64,10 +64,10 @@ But be aware: there might be **migrations** that are applied to your accounts da
 
 If you already have a core git checkout, you can skip the first step.
 
-1. clone the core repo, right next to your desktop repo folder: `git clone git@github.com:deltachat/deltachat-core-rust.git`
+1. clone the core repo, right next to your desktop repo folder: `git clone git@github.com:chatmail/core.git`
```

Comment:
to not break our own workflow, could we tell git to clone it into the old folder name?

Reply:
```suggestion
1. clone the core repo, right next to your desktop repo folder: `git clone git@github.com:chatmail/core.git deltachat-core-rust`
```